ui/public/i18n/language.json

```diff
@@ -513,6 +513,7 @@
         "signature": "Enable DKIM signing for outgoing messages to prevent spoofing.",
         "acls": "Type to search a user or group for ACLs.",
         "bayes_title": "Bayes filter training status.",
+        "virus_scan_only_attachments": "Scan mime parts of attachments separately, otherwise the complete mail will be transferred to the antivirus.",
```

Comment:
Scan attachments separately instead of whole mail. If checked, antivirus will be LESS effective, especially against phishing.

Reply:
Ok for changing the label, but please do not use `LESS` in uppercase

Reply:
done